src/librustc/mir/interpret/mod.rs

```diff
@@ -349,17 +349,19 @@ pub enum GlobalAlloc<'tcx> {
     Memory(&'tcx Allocation),
 }
 
+#[derive(Clone)]
```

Comment:
Semantic change I have no context for.

Reply:
This is no longer needed in fact, so I'll remove it.